bigquery/google/cloud/bigquery/bqdb/connection.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""Connection for the Google BigQuery DB-API."""
+
+from google.cloud import bigquery
+from google.cloud.bigquery.bqdb import cursor
+
+
+class Connection(object):
+    """Connection to Google BigQuery.
```

Comment:
I'd prefix all of these with `DB-API`, e.g. `DB-API Connection ...`, just for the sake of people who might search and land here.